code/modules/projectiles/projectile/special.dm

```diff
@@ -46,7 +46,7 @@
 	name = "freeze beam"
 	icon_state = "ice_2"
 	damage = 0
-	damage_type = BURN
+	damage_type = COLD

```

Comment:
>I've also added the new ""damage type"" (it's not actually a damage type per se but it can be passed to apply_damage and can be used as an object's damage type (I think)) COLD that will apply burn damage based multiplied by the species' coldmod. This causes a highly situational balance change in which species with a higher burnmod than coldmod (shadowlings notably) will take slightly less damage in a cold environment, while species with a higher coldmod than burnmod (do they exist?) will take more.

It applies burn damage, it's simply affected by coldmod. COLD can be passed as an argument to apply_damage().